org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -1321,7 +1348,8 @@ class CodeActionContext {
  * These commands are typically code fixes to either fix problems or to beautify/refactor code.
  */
 @JsonRpcData
-class CodeActionParams {
+class CodeActionParams extends WorkDoneProgressAndPartialParams {
```

Comment:
Nit: Should the type be called `WorkDoneProgressAndPartialResultParams` instead of `WorkDoneProgressAndPartialParams`?